card/spec/lib/card/reference_spec.rb

```diff
@@ -47,17 +47,21 @@
     newcard("Submarine","[[Yellow]]")
     newcard("Sun","[[Yellow]]")
     newcard("Yellow")
-    expect(Card["Yellow"].referencers.map(&:name).sort).to eq(%w{ Banana Submarine Sun })
+    yellow_refs = Card["Yellow"].referencers.map(&:name).sort
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.